docs/src/apps/jsonrpc-api.md

```diff
@@ -769,7 +770,35 @@ An RpcResponse containing a JSON object consisting of a string blockhash and Fee
 curl -X POST -H "Content-Type: application/json" -d '{"jsonrpc":"2.0","id":1, "method":"getRecentBlockhash"}' http://localhost:8899
 
 // Result
-{"jsonrpc":"2.0","result":{"context":{"slot":1},"value":{"blockhash":"CSymwgTNX1j3E4qhKfJAUE41nBWEwXufoYryPbkde5RR","feeCalculator":{"burnPercent":50,"lamportsPerSignature":5000,"maxLamportsPerSignature":100000,"minLamportsPerSignature":5000,"targetLamportsPerSignature":10000,"targetSignaturesPerSlot":20000}}},"id":1}
+{"jsonrpc":"2.0","result":{"context":{"slot":1},"value":{"blockhash":"CSymwgTNX1j3E4qhKfJAUE41nBWEwXufoYryPbkde5RR","feeCalculator":{"lamportsPerSignature":5000}}},"id":1}
+```
+
+### getRecentBlockhashLastValidSlot
```

Comment:
I'm definitely up for a shorter name, although I'm not sure `getFees` really describes what you're getting. I'll sleep on it...

Reply:
My thought there is that a blockhash and its expiration slot are both necessary means of qualifying the fee parameters. That is, if you send a signed transaction including the given blockhash by the given slot, the transaction fee is guaranteed to be determined by the given fee parameters. Deterministic blockhash expiration turns out to be a bonus feature.

Reply:
Yeah, that makes some sense, and I didn't think of anything better overnight. (I was toying with `getRecentFees` for a bit, but I think I mostly liked it because it would land next to `getRecentBlockhash` in the docs.) I'll push `getFees` and see how it looks